.github/workflows/spack_cpu_build.yaml

```diff
@@ -129,7 +152,7 @@ jobs:
                 padded_length: False
             mirrors:
               local-buildcache: oci://${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
-              spack: https://binaries.spack.io/develop
+            # spack: https://binaries.spack.io/develop
```

Comment:
Have to avoid using the public spack binary mirror for now due to https://github.com/spack/spack/issues/43919